City-of-Helsinki/drupal-helfi-platform-config
# UHF-X Views configuration translations

## What was done


* Updated every view and view translation via UI and then exported configuration

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-X_views_translations`
* Run `make drush-updb drush-cr`

## How to test


* [ ] After the instance update is finished, run `make drush-cex` and go through each configuration. Check that the configuration change makes sense (I.e compare the change to configuration changes in this PR)
* [ ] Check that the updated feature still works as before
